There is a bug I think it needs an update or whats app did soemthing...happens from time to time

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

There is a bug I think it needs an update or whats app did soemthing...happens from time to time

### Expected behavior

There is a bug I think it needs an update or whats app did soemthing...happens from time to time

### Steps to Reproduce the Bug or Issue

There is a bug I think it needs an update or whats app did soemthing...happens from time to time

### Relevant Code

There is a bug I think it needs an update or whats app did soemthing...happens from time to time

### Browser Type

Chromium

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

There is a bug I think it needs an update or whats app did soemthing...happens from time to time

### Additional context

There is a bug I think it needs an update or whats app did soemthing...happens from time to time

found 2 vulnerabilities (1 moderate, 1 high) in 307 scanned packages
  2 vulnerabilities require manual review. See the full report for details.
PS C:\Users\kshak\Desktop\WAA-main> npm install github:alechkos/whatsapp-web.js#fix-findImpl
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/alechkos/whatsapp-web.js.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\kshak\AppData\Roaming\npm-cache\_logs\2023-08-10T03_57_43_057Z-debug.log

--- a/src/util/Injected.js
+++ b/src/util/Injected.js
@@ -80,6 +80,12 @@ exports.ExposeStore = (moduleRaidStr) => {
         };
     }
 
+    if (window.mR.findModule('ChatCollection')[0] && window.mR.findModule('ChatCollection')[0].ChatCollection) {
+        if (typeof window.mR.findModule('ChatCollection')[0].ChatCollection.findImpl === 'undefined' && typeof window.mR.findModule('ChatCollection')[0].ChatCollection._find != 'undefined') {
+            window.mR.findModule('ChatCollection')[0].ChatCollection.findImpl = window.mR.findModule('ChatCollection')[0].ChatCollection._find;
+        }
+    }
+
     // TODO remove these once everybody has been updated to WWebJS with legacy sessions removed
     const _linkPreview = window.mR.findModule('queryLinkPreview');
     if (_linkPreview && _linkPreview[0] && _linkPreview[0].default) {

